html/user/server_status.php

```diff
@@ -187,9 +187,8 @@ function show_status_html($x) {
     end_table();
     
     global $server_version;
-    global $git_commit;
-    if ( isset($server_version) && isset($git_commit) ) {
-        echo "Server version: $server_version (<a href=https://github.com/BOINC/boinc/commit/$git_commit>$git_commit</a>)<br>";
+    if ( isset($server_version) ) {
+        echo "Server version: $server_version <br>";
```

Comment:
Since you now added `--abbrev=0` this can't be considered the "Server version" anymore. It's rather "Based on server version:".

Reply:
I don't understand. Please can you explain.

Reply:
See our long mail discussion: the `git describe` you now use just gives the *latest common* server release tag used by the server, it doesn't represent the server's version itself. Thus I recommend a more precise label for that version which really describes what it is.

Reply:
ok so ...
```
lfield:~/boinc (server_release/1.0) $ git describe
server_release/1.0/1.0.0-3-g19a64836a4
```
and 

```
lfield:~/boinc (server_release/1.0) $ git describe --abbrev=0 
server_release/1.0/1.0.0
```
git log shows the following (some output suppressed)
```

commit 19a64836a416268ee8c9ca67fe3340efcbacbc0f (HEAD -> server_release/1.0, tag: server_release/1.0/1.0.1, origin/server_release/1/1.0)
...
commit cefcf545b825504571e080ff35578508907e679e
...
commit 771620daa7abe057a7eba030650086866a46c5c1
...
commit e72c05ace220190c82550e9f91745590fb2d8560 (tag: server_release/1.0/1.0.0)
```
So my guess is that using `git describe` without `--abbrev=0 `appends -3-g19a64836a4 where this is the number of commits since the tag and the last commit. So your argument is that as the branch may be few changes ahead if we made and local changes, we should use the words _based on_ ?

Reply:
Actually, if you are restricting the server version to *official* (annotated) tags, a link to the official repository might be useful (`https://github.com/BOINC/boinc/releases/tag/server_release%2F1.0%2F1.0.1` for the tag or `https://github.com/BOINC/boinc/tree/server_release/1.0/1.0.1` directly to the code).

Reply:
I think I would still use '_server version_'. After installation, files can get hot fixed, modified by configuration management systems or tweaked manually. If we do want to make this explicit, there is probably a better word to use like series.

Reply:
ok. decided between the two of you if you want a link or not and let me know.

Reply:
@bema-aei wants to use a fixed URL to github/boinc, not a dynamically generated one from the actual repo's remote. That's fine with me, as long as you clearly refer to the `latest common server release tag` only, that is, use `--abbrev=0`.

Reply:
> I think I would still use 'server version'.

The reason I don't like this is that 'server version' is ambiguous. It doesn't tell the reader whether that refers to upstream BOINC or the project's deployment.

Reply:
I have just done this. Please check. Can we agree on the text? Does it need to be changed from server version?

Reply:
Can you think of anything better than based on? Upstream version? Powered by ? Built from?

Reply:
"Upstream version" and "Powered by" are both fine with me 👍

Reply:
What about _server release_? This suggests something a little more general.

Reply:
Upstream server release?